mlflow/go/pkg/server/server.go

```diff
@@ -158,7 +155,7 @@ func newAPIApp(cfg *config.Config) (*fiber.App, error) {
 		return nil, err
 	}
 
-	mlflowService, err := NewMlflowService(cfg)
+	mlflowService, err := NewMlflowService(logrus.StandardLogger(), cfg)
```

Comment:
I'd say we either need to bubble the logger parameter passing all the way up to the launcher, or if we're just using `logrus.StandardLogger()` anyway we can simply pass it to the Gorm logger adapter.

Reply:
Done, passing the logger from the entry point now.